PlaygroundApp/PlaygroundApp/Menu/MenuViewController.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  MenuViewController.swift
+//  PlaygroundApp
+//
+//  Created by Learning on 16/03/23.
+//
+
+import UIKit
+
+class MenuViewController: UIViewController {
+    
+    @IBOutlet weak var collection: UICollectionView!
+    @IBOutlet weak var table: UITableView!
+    
+    struct Menu {
+        let image: String
+        let title: String
+        let rating: String
+        let price: String
+    }
+    
+    let data: [Menu] = [
+        Menu(image: "americano", title: "Americano", rating: "3", price: "Rp.18.0000"),
+        Menu(image: "pancake-blueberry", title: "Pancake Blueberry", rating: "5", price: "Rp.38.000"),
+        Menu(image: "caesar-salad", title: "Caesar Salad", rating: "4", price: "Rp.40.000"),
+        Menu(image: "mango-juice", title: "Mango Juice", rating: "4", price: "Rp.36.000"),
+    ]
+    
+    let categoryList: [String] = ["Favorit", "Semua", "Makanan", "Minuman"]
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        table.dataSource = self
+        table.register(UINib(nibName: "MenuViewCell", bundle: nil), forCellReuseIdentifier: "MenuCell")
+        
+        collection.dataSource = self
+        collection.register(UINib(nibName: "MenuCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "MenuCollectionCell")
+    }
```

Comment:
ini juga untuk mengurangi human error bisa pakai ini:
`        collection.register(UINib(nibName: String(describing: MenuCollectionViewCell.self), bundle: Bundle(for: MenuCollectionViewCell.self)), forCellWithReuseIdentifier: String(describing: MenuCollectionViewCell.self))
`